src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -34,7 +34,7 @@ export const ProfileFestsRoute = () => {
   });
 
   function openSlopFestModal() {
-    openModal(<SlopFestModal onClose={closeModal} />);
+    openModal(<SlopFestModal buttonTitle="Submit" onClose={closeModal} />);
```

Comment:
Hey Danny! It would probably be best to update this button text to say "Fest On!" to match the figma design.

Also, just a suggestion to move the buttonTitle down to line 214 instead so that it's passed as a prop to Form.Submit instead. This matches the other modal setups as well if that helps!

Reply:
Change made! I was looking at the wrong Figma design option on the left and didn't see the slop fest modal, so I had made up my own. Thanks!